### ISSUE TITLE
package command line option does not affect lexer

### ISSUE THREAD

**Issue by KvanTTT** [pboyer/antlr4#46]
I am using command line option `-package main`. But it's working only for parser, not for lexer.

**Comment by sridharxp** [comment]
In Go.stg (line 1330) the following change seems to work. Create the jar file
from
`package parser`
to

```
<if(lexerFile.genPackage)>
package <lexerFile.genPackage> 
<else>
package parser 
<endif>

```

**Comment by pboyer** [comment]
Fixed by #49. @KvanTTT Let me know if you still see this issue.

**Comment by KvanTTT** [comment]
It's working for now. Thanks!
